src/main/scala/net/cakesolutions/CakePlatformPlugin.scala

```diff
@@ -1,50 +1,57 @@
 // Copyright: 2017 https://github.com/cakesolutions/sbt-cake/graphs
 // License: http://www.apache.org/licenses/LICENSE-2.0
-package net.cakesolutions
 // Copyright 2015 - 2016 Sam Halliday (derived from sbt-sensible)
```

Comment:
I certainly have no intention of removing a copyright notice - on the contrary, having had work plagiarised by *major* corporations, I think it is absolutely essential that all due diligence is given to all authors and copyright holders of code.

Now, having actually configured and checked the behaviour of sbt-cake's use of `sbt-header`, I'm able to say that I didn't find your concerns here to be valid - i.e. both copyright notices (with a line separator) can happily reside at the top of the file with no untoward issues being detected.